.changeset/violet-trees-play.md

```diff
@@ -0,0 +1,5 @@
+---
+'@backstage/plugin-shortcuts': patch
```

Comment:
Now something has been added to the interface, this is a breaking change to this library, so this should be `minor` here, as we're on `<1.0.0` 👍